src/ask-password/ask-password.c

```diff
@@ -117,15 +123,19 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_ECHO:
-                        arg_flags |= ASK_PASSWORD_ECHO;
+                        SET_FLAG(arg_flags, ASK_PASSWORD_ECHO, true);
+                        break;
+
+                case ARG_EMOJI:
+                        emoji = optarg;
                         break;
 
                 case ARG_NO_TTY:
-                        arg_flags |= ASK_PASSWORD_NO_TTY;
+                        SET_FLAG(arg_flags, ASK_PASSWORD_NO_TTY, true);
```

Comment:
please don't do this. SET_FLAG() is useful when the third argument is a variable expression, but here you pass `true` only, i.e. a fixed constant and literal.

i.e. the idea of SET_FLAG() is that it can both turn off and turn on bits. But if you just want to turn the off, the simple arg_flags |= … expression is simpler and nicer, and equivalent.